validate/setup-guides/do/sgDroplet.md

```diff
@@ -79,30 +87,34 @@ This is going to restrict access to your Node from this location only (your `loc
 The **less restrictive** your access, the more **vulnerable** your Node becomes.
 :::
 
+### Add Secure Shell rules
+
 Let's walk through the **two** Inbound Rules.
 
-We will leave **SSH** as our Type, and then highlight and click the `X` to remove both the `All IPv4` and `All IPv6` items.
+We will leave **SSH** as our Type, and then highlight (select the individual item) and click the `X` to remove both the `All IPv4` and `All IPv6` items.
 
 ![](/img/validator_nodes/do-sg7.png)
 
 
-Add to that box in its place (replacing the `All IPv4` and `All IPv6`) the **IP address** we retrieved from [here](https://www.whatismyip.com). One small modification: we want to add `/32` to the end. 
+Add to that box in its place (replacing the (now removed) `All IPv4` and `All IPv6`) the **IP address** we retrieved from [here](https://www.whatismyip.com). One small modification: we want to add `/32` to the end. 
 
-if `111.111.111.111` was our IP address then we would add in `111.111.111.111/32`
+Example) If `111.111.111.111` was our IP address then we would add in `111.111.111.111/32`
 
 :::note
-If you want to access your Node from other locations, you will have to add the **IP addresses** from those locations in `one-by-one`. If you are more advanced, you can add `subnets` (out of scope of this tutorial). 
+If you want to access your Node from other locations, you will have to add the **IP addresses** from those locations in `one-by-one`. If you are more advanced, you can add `subnets` by updating the `CIDR` block (subnet mask) (out of scope of this tutorial). 
```

Comment:
There are too many info/note/warning callouts on this page. If everything is extra important then nothing is. Could you clean it up to just a few callouts with none of them stacked one after another?